libraries/common/load-vue-components.js

```diff
@@ -17,19 +17,22 @@ export default (filenames) =>
           const dom = new DOMParser().parseFromString(text, "text/html");
           const css = dom.querySelector("style")?.textContent;
           if (css) {
-            if (chrome.runtime.getManifest().version_name.includes("-prerelease")) {
-              const normalizedCss = css.replace("\n", "").trimEnd();
-              const normalizedText = text.replace(/\r/g, "");
-              const cssFirstLine = normalizedCss.substring(0, normalizedCss.indexOf("\n"));
-              const linesToAdd = normalizedText.split("\n").findIndex((line) => line === cssFirstLine);
-              const newLines = Object.assign(Array(linesToAdd), {
-                0: "/*",
-                [linesToAdd - 1]: "<style> */",
-              }).join("\n");
-              styles[filename] = `${newLines}\n${normalizedCss}/* \n</style> */\n/*# sourceURL=${htmlUrl} */`;
-            } else {
-              styles[filename] = css;
-            }
+            chrome.management.getSelf((result) => {
```

Comment:
check this one time at the top of the file. it should never change without the entire extention reloading.